test/dotnet.Tests/CommandTests/Run/RunFileTests.cs

```diff
@@ -446,6 +446,42 @@ Hello from App
                 """);
     }
 
+    [Fact]
+    public void ProjectInCurrentDirectory_NoRunVerb()
+    {
+        var testInstance = _testAssetsManager.CreateTestDirectory();
+        Directory.CreateDirectory(Path.Join(testInstance.Path, "file"));
+        File.WriteAllText(Path.Join(testInstance.Path, "file", "Program.cs"), s_program);
+        Directory.CreateDirectory(Path.Join(testInstance.Path, "proj"));
+        File.WriteAllText(Path.Join(testInstance.Path, "proj", "App.csproj"), s_consoleProject);
+
+        new DotnetCommand(Log, "../file/Program.cs")
+            .WithWorkingDirectory(Path.Join(testInstance.Path, "proj"))
+            .Execute()
+            .Should().Pass()
+            .And.HaveStdOutContaining("""
+                Hello from Program
+                """);
```

Comment:
So the fact that there is no "echo args" in the output means we ran the file rather than running the project? Is that right?

Reply:
The project cannot even run, it has no source files (only a project file). If it could run, it should print `Hello from App` also. And yes, there would be also `echo args`.